packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -324,8 +326,8 @@ export interface DDSFuzzSuiteOptions {
 	 * an attach op.
 	 */
 	detachedStartOptions: {
-		attachProbability: number;
 		enabled: boolean;
+		numOpsBeforeAttach: number;
```

Comment:
I think the scenarios that `enabled: false` powered before can be replaced with `numOpsBeforeAttach: 0`. Might be a nice simplification of the API.